bin/tools.py

```diff
@@ -447,7 +456,9 @@ def build_parser():
         action='store_true',
         help='Rerun all generators to make sure generators are deterministic.',
     )
-    genparser.add_argument('--timeout', '-t', type=int, help='Override the default timeout.')
+    genparser.add_argument(
+        '--timeout', '-t', type=int, help='Override the default timeout (Default: 30).'
```

Comment:
`Override the default timeout. Default: 30` for consistency, here and elsewhere.

Independent of that, should we specify the unit somewhere? `default timeout in seconds` maybe? Although that's probably obvious and more verbose

Reply:
Check, I've checked for consistency for "Default" clauses in the command descriptions :slightly_smiling_face: (57be5a8)

I don't think the unit is necessary here, "seconds" should™ be obvious :stuck_out_tongue: